src/muqsit/pmarmorstand/entity/ArmorStandEntity.php

```diff
@@ -128,12 +128,16 @@ public function saveNBT() : CompoundTag{
 		$nbt = parent::saveNBT();
 
 		$armor_pieces = [];
-		foreach($this->getArmorInventory()->getContents() as $slot => $item){
-			$armor_pieces[] = $item->nbtSerialize($slot);
+		foreach($this->getArmorInventory()->getContents() as $slot => $item) {
+			if (!$item->isNull()) {
```

Comment:
This check isn't needed because `Inventory::getContents()` (unlike `Inventory::getContents(true)`) always returns non-null items